### DISCUSSION TITLE
DataPortal issue - Invalid operation - Create is not allowed

### DISCUSSION THREAD

**Discussion by sylum** [MarimerLLC/csla#1825] [Questions] [answered]
**Question**

Hi Folks,

I’m facing a DataPortal issue when i made a CommandBase call.
Actually, I get the following exception :
```
DataPortal.Create failed on server
Invalid operation - Create is not allowed
```

The issue can be reproduce with the projecttracker (for whatever UI techno : Bazor, Wpf, Xamarin) when calling the **ProjectExistsCommand** or **ResourceExistsCommand**.

By the way, all the other CSLA stereotypes  (BusinessBase, BusinessListBase...) work well. I have been able to fetch, update and detele record into database using csla stereotypes.

I have also try to up to date the CSLA libraries version but same issue.
and same issue when identified (as manager or admin)

Thanks in advance for your help,

Cedric

**Version and Platform**
CSLA version: 5.3.1
OS: Windows10,
Platform: Xamarin Forms 4.8.0.1451 / Android

**Comment by EricNgo1972** [comment, marked as answer]
Try replace constructors:

```c#
public ResourceExistsCommand() 
public ResourceExistsCommand(int id)
```
to 

```c#
[Create, RunLocal]
private void Create() 

[Create, RunLocal]
private void Create(int id)

```

**Reply by rockfordlhotka** [reply]
That's correct. I wonder if I missed some necessary updates in ProjectTracker 🤔

**Comment by sylum** [comment]
Thanks Tung Ngo, Thanks Rocky, the solution works like a charm.
I guess i still need to learn my CSLA since i'm using the version 5
